pyartcd/pyartcd/pipelines/prepare_release.py

```diff
@@ -176,6 +176,9 @@ async def run(self):
                         # Connect advisory to the batch_id
                         change_advisory_batch(advisory_id=advisories[ad], batch_id=batch_id)
                         _LOGGER.info(f"Advisory {advisories[ad]} connected to the batch id {batch_id}")
+                    else:
+                        change_advisory_batch(advisory_id=advisories[ad], batch_id=batch_id, clear_batch=True)
```

Comment:
why pass in batch_id=batch_id if we want to clear it?

Reply:
Let's make that param optional

Reply:
updated